index.html

```diff
@@ -45,5 +45,138 @@ <h2>LET’S CONNECT</h2>
+        </section>
+        <section id="about">
+            <h2 class="header">About Myself</h2>
+            <p class="paragraph">
+                Hello I’m a software developer! I can help you build a product, feature or website. 
+                Look through some of my work and experience! If you like what you see and have a project 
+                you need coded, don’t hesitate to contact me.
+            </p>
+            <h2 class="header-2">LET’S CONNECT</h2>
+            <ul class="social-icon social-icon-2">
+                <li><a href="#"><img src="./assets/github.svg" alt="Twitter"></a></li>
+                <li><a href="#"><img src="./assets/Linkedin.svg" alt="LinkedIn"></a></li>
+                <li><a href="#"><img src="./assets/angellist.svg" alt="Medium"></a></li>
+                <li><a href="#"><img src="./assets/twt.svg" alt="GitHub"></a></li>
+                <li><a href="#"><img src="./assets/meduim.svg" alt="AngelList"></a></li>
+            </ul>
+            <button type="button" id="reume">Get my resume</button>
+            <ul id="tech">
+                <li>
+                    <h2 class="tech dropdown">Languages</h2>
+                    <ul id="languages">
+                        <li>
+                            <img src="./assets/js.svg" alt="js">
+                            <h3>JavaScript</h3>
+                        </li>
+                        <li>
+                            <img src="./assets/html.svg" alt="js">
+                            <h3>HTML</h3>
+                        </li>
+                        <li>
+                            <img src="./assets/css.svg" alt="js">
+                            <h3>CSS</h3>
+                        </li>
+                    </ul>
+                </li>
+                <li>
+                    <h2 class="tech underline">Frameworks</h2>
+                </li>
+                <li>
+                    <h2 class="tech underline">Skills</h2>
+                </li>
+            </ul>
+        </section>
+        <section class="white-bg">
+            <article id="contact-form">
+                <h2 class="header white">Contact me</h2>
+                <p class="paragraph white">
+                    If you have an application you are interested in developing, a feature that you need built 
+                    or a project that needs coding. I’d love to help with it!
+                </p>
+                <form class="form" action="https://formspree.io/f/mnqywewj" method="POST">
+                    <input type="text" name="name" maxlength="30" placeholder="Lorem Ipsum" required>
+                    <input type="email" name="email" placeholder="loremipsum@gmail.com" required>
+                    <textarea name="message" placeholder="Write your message here..."  maxlength="500" required></textarea>
+                    <input id="submit" type="submit" value="Get in touch">
+                  </form>
+            </article>
+        </section>
```

Comment:
- [ ] Your form functions well and appears nice only that the `background-image` is substantively different from the user stories - project requirement (Figma wireframe). I have attached a comparison for your notice below:

|Your Design| Figma Design|
|:--------------:|:-----------------:|
|![image](https://user-images.githubusercontent.com/99715640/252774506-7ddb38d1-bfcd-4b74-98fb-488d90450e6b.png)|![image](https://user-images.githubusercontent.com/99715640/252774811-8fa0846f-0019-43ba-8ee0-07f2868c1498.png)|

- [ ] Kindly ensure that all elements in the background are placed as seen in Figma's design. These have great implications for the user experience on your page.
- [ ] Also, I noticed that your `Get in touch` button does not have `:hover` and `:active` states as your `Get resume` has. It would be nice if you can change the color properties (hover and active) of other buttons i.e. `See project` to that of the `Get Resume` which is the approved one and for uniformity's sake.
- [ ] Lastly, if you check that image well, you will see a red arrow pointing at the base of your form. Seems like you have an overflow on your y-axis resulting to some inconvenient side scroll before abstracting well with your design. Kindly remove this.